project/profiles/capacity4more/modules/c4m/content/c4m_content_wiki_page/c4m_content_wiki_page.features.field_base.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @file
+ * c4m_content_wiki_page.features.field_base.inc
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function c4m_content_wiki_page_field_default_field_bases() {
+  $field_bases = array();
+
+  // Exported field_base: 'field_c4m_editing_open'.
+  $field_bases['field_c4m_editing_open'] = array(
```

Comment:
@soniCaH I'm also not sure about the need for another table - as `{node_access}` allows you also to grant for user. Basically we could have any arbitrary real attached to any (arbitrary) ID. 
In all previous ACLs we've implemented we always used just the `{node_access}`.

Unless I'm of course missing your point :)